README.md

```diff
@@ -105,12 +105,17 @@ Once you've successfully acquired a token, you can run authenticated commands wi
 
 ### Unattended build agents 
 
-With Azure DevOps Pipelines, please use the [NuGet Authenticate](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/package/nuget-authenticate?view=azure-devops) task before running NuGet, dotnet or MSBuild commands that need authentication.
+#### Azure DevOps Pipelines
+Use the [NuGet Authenticate](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/package/nuget-authenticate?view=azure-devops) task before running NuGet, dotnet or MSBuild commands that need authentication.
 
+#### Other automated build scenarios
 If you're running the command as part of an automated build on an unattended build agent outside of Azure DevOps Pipelines, you can supply an access token directly using the `VSS_NUGET_EXTERNAL_FEED_ENDPOINTS` [environment variable](#environment-variables). The use of [Personal Access Tokens](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops) is recommended. You may need to restart the agent service or the computer before the environment variables are available to the agent.
 
 ### Docker containers
-[Sample Dockerfile](https://github.com/microsoft/artifacts-credprovider/blob/master/samples/dockerfile.sample.txt)
+[Managing NuGet credentials in Docker scenarios](https://github.com/dotnet/dotnet-docker/blob/master/documentation/scenarios/nuget-credentials.md#using-the-azure-artifact-credential-provider)
+
+### Azure DevOps Server
+The Artifacts credential provider cannot cannot acquire credentials interactively for an on premise Azure DevOps Server, therefore, the VSS_NUGET_EXTERNAL_FEED_ENDPOINTS environment variable must be used as an alternative. Supply a [Personal Access Token](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops) directly using the `VSS_NUGET_EXTERNAL_FEED_ENDPOINTS` [environment variable](#environment-variables). From Azure DevOps Server 2020 RC1 forward, the NuGet Authenticate task can be used in Pipelines. 
```

Comment:
```suggestion
The Artifacts credential provider cannot cannot acquire credentials interactively for an on-premises Azure DevOps Server. In many cases, authentication will work without the credential provider using the Negotiate/NTLM/Kerberos schemes. In other cases, artifacts-credprovider and the VSS_NUGET_EXTERNAL_FEED_ENDPOINTS environment variable may be used as an alternative. Supply a [Personal Access Token](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops) directly using the `VSS_NUGET_EXTERNAL_FEED_ENDPOINTS` [environment variable](#environment-variables). From Azure DevOps Server 2020 RC1 forward, the NuGet Authenticate task can be used in Pipelines. 
```

Reply:
"on premise" -> "on-premises", and adding a sentence mentioning that artifacts-credprovider may not be necessary

Reply:
Thanks! Done